### PR TITLE
[search] Heuristics for rating.

### PR DESCRIPTION
**Pull request by tatiana-yan**
новые бубны

### CODE REVIEW

**Comment by tatiana-yan** [comment]
JTALL

**Comment by tatiana-yan** [comment]
JTALL

File: search/ranking_info.cpp [hunk 1]
```diff
@@ -48,6 +53,20 @@ double TransformDistance(double distance)
 {
   return min(distance, RankingInfo::kMaxDistMeters) / RankingInfo::kMaxDistMeters;
 }
+
+double TransformRating(pair<uint8_t, float> const & rating)
+{
+  double r = 0.0;
+  // From satistics.
```
Comment (mpimenov) [inline search/ranking_info.cpp]: s/satistics/statistics/

File: search/ranking_info.cpp [hunk 2]
```diff
@@ -75,6 +95,7 @@ string DebugPrint(RankingInfo const & info)
   os << "m_distanceToPivot:" << info.m_distanceToPivot;
   os << ", m_rank:" << static_cast<int>(info.m_rank);
   os << ", m_popularity:" << static_cast<int>(info.m_popularity);
+  os << ", m_rating: [" << static_cast<int>(info.m_rating.first) << ", "<< info.m_rating.second << "]";
```
Comment (mpimenov) [inline search/ranking_info.cpp]: clang-format

File: search/ranking_info.hpp [hunk 3]
```diff
@@ -36,6 +37,9 @@ struct RankingInfo
   // Popularity rank of the feature.
   uint8_t m_popularity = 0;
 
+  // UGC rating and confidence.
```
Comment (mpimenov) [inline search/ranking_info.hpp]: Confidence and UGC rating.

File: search/ranking_info.cpp (hunk 1 above)
Comment (tatiana-yan) [reply search/ranking_info.cpp]: done

File: search/ranking_info.cpp (hunk 2 above)
Comment (tatiana-yan) [reply search/ranking_info.cpp]: done

File: search/ranking_info.hpp (hunk 3 above)
Comment (tatiana-yan) [reply search/ranking_info.hpp]: done

**Pull request merged**